recipe/meta.yaml

```diff
@@ -12,7 +12,8 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  noarch: python
```

Comment:
This requires a [re-rendering]( https://github.com/conda-forge/conda-smithy#re-rendering-an-existing-feedstock ) to fix up the feedstock (e.g. cutdown on CIs and update the README). Could you please submit one in a new PR?

Reply:
Hi, not exactly sure what change is requested? Also this is an older version of my package, not so sure if I want to push it up?

Reply:
Am requesting that the feedstock be re-rendered.

Reply:
Should add have only commented on this diff to draw attention to the fact that [adding `noarch: python` requires a re-rendering]( https://conda-forge.org/docs/meta.html#building-noarch-packages ). Though the re-rendering can and should be made using the latest commit of this recipe. Please let me know if there are other questions.

Reply:
ok, noted, have re-rendered on the latest release PR #12 .

Reply:
Thanks.